test/Microsoft.TestPlatform.CommunicationUtilities.UnitTests/Serialization/TestCaseSerializationTests.cs

```diff
@@ -28,6 +28,8 @@ public class TestCaseSerializationTests
                                                    Traits = { new Trait("Priority", "0"), new Trait("Category", "unit") }
                                                };
 
+        #region v1 Tests
```

Comment:
Require test where a TestCase doesn't have any properties and can be serialized properly.